http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -436,7 +436,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
                                                         params.runtime.auditInfo.creator
           )
 
-          createDiskJobId = WsmJobId(s"create-disk-${disk.id.toString.take(10)}")
+          createDiskJobId = WsmJobId(s"${UUID.randomUUID()}")
```

Comment:
maybe make a follow-up to make this case class take a UUID?

Reply:
yeah that makes sense, I can add it to the [this spike](https://broadworkbench.atlassian.net/browse/IA-4850)